mobify/platform-scaffold
# Update redux store if a webview logs in

This depends on https://github.com/mobify/platform-scaffold/pull/466

 **JIRA**: https://mobify.atlassian.net/browse/HYB-1007
 **Linked PRs**: https://github.com/mobify/platform-scaffold/pull/466

## Changes
- Refresh the cart when a user logs in

## How to test-drive this PR
- Change `package.json` to point Astro to`feature/topplugin-rpc` 
- Run `npm run deps` in the `native` folder
- Change `baseConfig.js` in `native/config` to enable preview if disabled
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Go to account
- Log in with an account with items in their cart
- Go to a different tab
- Cart icon should show the same number of items as the accoutn tab
